npm-shrinkwrap.json

```diff
@@ -1,9 +1,495 @@
 {
   "name": "fxa-content-server",
-  "version": "1.110.1",
+  "version": "1.110.2",
```

Comment:
There's no need to update shrinkwrap if you're not adding or removing dependencies, so you can undo your changes for this file and keep the diff focused on just what we we're concerned with.